spec/features/user_auth_spec.rb

```diff
@@ -1,49 +1,81 @@
 require 'rails_helper'
 
 RSpec.describe "the user authorization feature", type: :feature do
-  let(:valid_user) { User.create(email: "paul@mail.com", full_name: 'Paul', password: "paul") }
-  let(:other_user) { User.create(email: "lori@mail.com", full_name: 'Lori', password: "lori") }
-  let(:admin) { User.create(email: "admin_man@mail.com", full_name: 'AdminMan', password: "admin", role: 'admin') }
+  context 'when a default user signs in' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.